holidays/countries/bonaire_sint_eustatius_and_saba.py

```diff
@@ -0,0 +1,126 @@
+#  holidays
+#  --------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see CONTRIBUTORS file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+
+from gettext import gettext as tr
+
+from holidays.calendars.gregorian import JUL, AUG, SEP
+from holidays.groups import ChristianHolidays, InternationalHolidays
+from holidays.observed_holiday_base import ObservedHolidayBase, SUN_TO_PREV_SAT
+
+
+class BonaireSintEustatiusAndSaba(ObservedHolidayBase, ChristianHolidays, InternationalHolidays):
+    """Bonaire Sint Eustatius And Saba holidays.
+
+    References:
+        * [Public Holidays 2025](https://web.archive.org/web/20250620050728/https://english.rijksdienstcn.com/social-affairs-work/employment/official-holidays)
+        * [Public Holidays in Sint Eustatius in 2025](https://web.archive.org/web/20250621132223/https://www.officeholidays.com/countries/bonaire-st-eustatius-saba/sint-eustatius/2025)
+        * [Upcoming Bonaire, St Eustatius and Saba Public Holidays](https://web.archive.org/web/20250427005333/https://www.qppstudio.net/public-holidays/bonaire__st_eustatius_and_saba.htm)
```

Comment:
```suggestion
    """Bonaire, Sint Eustatius and Saba holidays.

    References:
        * [Public Holidays 2025](https://web.archive.org/web/20250620050728/https://english.rijksdienstcn.com/social-affairs-work/employment/official-holidays)
```
Other sourced from PR comments need to be added instead.